### PR TITLE
added link to blueprints examples in README.md

### PR DESCRIPTION
**Pull request by AdarShaked**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,6 @@ See [AWS Plugin](http://docs.getcloudify.org/latest/plugins/aws/)
 
 # Requirements
 boto3 AWS Python Library version 1.9.57
+
+# Examples
+For more examples navigate to [cloudify blueprints examples github repository](https://github.com/cloudify-community/blueprint-examples/).
```
Comment (EarthmanT) [inline README.md]: Let's reword it to say:

```For official blueprint examples using this Cloudify plugin, please see [Cloudify Community Examples](...).```

How does that sound?

Comment (AdarShaked) [reply README.md]: great.
maybe  we should write: [Cloudify Community Blueprints Examples]
